### PR TITLE
combine instructions

### PR DESCRIPTION
**Pull request by loafyloaf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -159,14 +159,24 @@ services:
...
 ### 3. Start the MEAN stack on LinuxONE.
 
-```docker-compose up``` point your browser to ```http://[host ip]:4200``` and enjoy!
+Clone the repo and navigate to the correct directory
+```bash
+$ git clone https://github.com/IBM/Cloud-Native-Workloads-on-LinuxONE.git
+$ cd Cloud-Native-Workloads-on-LinuxONE/files/mean-docker
+```
+
+```docker-compose up -d``` point your browser to ```http://[host ip]:8080``` and enjoy!
```
Comment (animeshsingh) [inline README.md]: enjoy maybe is too casual here :-)

File: README.md [hunk 2]
```diff
@@ -159,14 +159,24 @@ services:
...
+The ``image:``` tag in the docker-compose.yml and the base image name ```FROM node:6``` for the [express-server/Dockerfile](https://github.com/gangachris/mean-docker/blob/master/express-server/Dockerfile) currently point to x86 version of the binaries. In order to run on the LinuxONE platform, we need to point them to the z architecture compatible binaries. A quick search on DockerHub will reveal [sinenomine/mongodb-s390x](https://hub.docker.com/r/sinenomine/mongodb-s390x/) and [s390x/ibmnode](https://hub.docker.com/r/s390x/ibmnode/). Just add a ```:latest``` tag and update the appropriate locations.
 
 ### 3. Start the MEAN stack on LinuxONE.
 
-```docker-compose up``` point your browser to ```http://[host ip]:4200``` and enjoy!
+Clone the repo and navigate to the correct directory
+```bash
+$ git clone https://github.com/IBM/Cloud-Native-Workloads-on-LinuxONE.git
+$ cd Cloud-Native-Workloads-on-LinuxONE/files/mean-docker
```
Comment (animeshsingh) [inline README.md]: You can mention go the root director of your cloned repo, and then cd /files/mean-docker

**Comment by animeshsingh** [comment]
Apart from 2 comments there, LGTM. @zInnovationLab take a look as well.

**Comment by zInnovationLab** [comment]
I'm fine with the updates. Still going to add some screenshots etc when I get cycles.

**Pull request merged**